doc/zmq_getsockopt.txt

```diff
@@ -551,6 +551,14 @@ blocking or dropping sent messages. Refer to the individual socket descriptions
 in linkzmq:zmq_socket[3] for details on the exact action taken for each socket
 type.
 
+The 'ZMQ_RCVHWM' and 'ZMQ_SNDHWM' options are logical extensions to one
```

Comment:
I am not sure what "logical extensions to one another" means. Maybe simply say "are closely related"?

Reply:
"Closely related" might be a better term. I can use that term instead.